src/take.js

```diff
@@ -20,13 +23,11 @@ var slice = require('./slice');
  * @return {Array}
  * @example
  *
- *      R.take(3,[1,2,3,4,5]); //=> [1,2,3]
- *
- *      var members= [ "Paul Desmond","Bob Bates","Joe Dodge","Ron Crotty","Lloyd Davis","Joe Morello","Norman Bates",
- *                     "Eugene Wright","Gerry Mulligan","Jack Six","Alan Dawson","Darius Brubeck","Chris Brubeck",
- *                     "Dan Brubeck","Bobby Militello","Michael Moore","Randy Jones"];
- *      var takeFive = R.take(5);
- *      takeFive(members); //=> ["Paul Desmond","Bob Bates","Joe Dodge","Ron Crotty","Lloyd Davis"]
```

Comment:
This was overdone, but I still like having this [joke][db] in the comments.  I think a lot of people will recognize an appreciate it, and an occasional humorous bit like this is to my mind a good thing<sup>TM</sup>.

```js
    var takeFive = R.take(5);
    var personnel = ['Dave Brubeck', 'Paul Desmond', 'Eugene Wright', 'Joe Morello', 
                     'Gerry Mulligan', 'Bob Bates', 'Joe Dodge', 'Ron Crotty'];
    var lineup = takeFive(personnel); 
    //=> ['Dave Brubeck', 'Paul Desmond', 'Eugene Wright', 'Joe Morello', 'Gerry Mulligan']
```

Of course maybe the jokes on me, since Mulligan wasn't in the line-up for Take Five.


  [db]: https://www.youtube.com/watch?v=vmDDOFXSgAs